docs/conf.py

```diff
@@ -36,16 +38,16 @@ class TypeMock(type):
     pass
 
 MOCK_MODULES = [
-    'numpy',
-    'numpy.dtype',
+    # 'numpy',
+    # 'numpy.dtype',
     'scipy',
     'scipy.special',
     'scipy.optimize',
     'scipy.stats',
     'cython',
     'thewalrus.libwalrus',
-    'numba',
-    'numba.jit'
+    # 'numba',
+    # 'numba.jit'
     ]
```

Comment:
There were originally several reasons why these modules are mocked out:

* There is a bug in Sphinx (it might be fixed now) where Sphinx-autodoc would mistakenly include NumPy function documentation in projects that simply import it. This happened in Strawberry Fields, hence why it is probably mocked out here. I think it's okay to un-mock it now.

* Mocking allows Sphinx to import the project and inspect docstrings without requiring the dependencies be installed. This is quite important for `'thewalrus.libwalrus'`, as we don't want to have to build the C++ code on the RTDs environment.

Reply:
I've noticed that `scipy` and `numpy` are now in the `doc/requirements.txt`, so are being installed on read the docs. So maybe SciPy doesn't have to be mocked any more either.